app/routes/_.$path.tsx

```diff
@@ -0,0 +1,10 @@
+import { PageBuilderRoute } from "./page-builder/page-builder-page";
+import { NotFound } from "~/components/not-found";
+
+export { loader } from "./page-builder/loader";
+
+export function CatchBoundary() {
```

Comment:
This `CatchBoundary()` I don't think is doing anything so we can remove it. I think the proper syntax is `ErrorBoundary()`